src-java/base-topology/base-messaging/src/test/java/org/openkilda/messaging/info/switches/v2/GroupsValidationEntryV2Test.java

```diff
@@ -37,9 +36,9 @@ public void splitAndUniteEmptyGroupEntryTest() {
                 .proper(new ArrayList<>())
                 .build();
         List<GroupsValidationEntryV2> list = entry.split(4, 4);
-        assertEquals(1, list.size());
+        Assertions.assertEquals(1, list.size());
```

Comment:
Just a question: can we just import `MatcherAssert.assertEquals` to
* do not change current test code
* remove unnecessary `Assertions.` from the code